e7-damage-calc/heroes.js

```diff
@@ -943,7 +943,7 @@ const heroes = {
         enhance: [0.05, 0.05, 0, 0.1, 0.1],
       },
       s2: {
-        rate: () => 1.5 + (elements.caster_nb_buff.value() > 1 ? (elements.caster_nb_buff.value()-1)*0.07 : 0),
+        rate: () => 1.5 + (elements.caster_nb_buff.value() > 1 ? (elements.caster_nb_buff.value())*0.07 : 0),
```

Comment:
Shouldn't the check for `elements.caster_nb_buff.value()` change from `> 1` to `> 0`? meaning we actually don't need the test anymore...

```suggestion
        rate: () => 1.5 + elements.caster_nb_buff.value()*0.07,
```